Dockerfile

```diff
@@ -77,10 +74,12 @@ RUN pip install -r /tmp/requirements-production.txt
 WORKDIR /code
 COPY . /code
 
+RUN pip install -e ".[all]"
+
 # Create instance
 RUN /code/scripts/create-instance.sh && \
-    chgrp -R 0 ${APP_INSTANCE_PATH} && \
-    chmod -R g=u ${APP_INSTANCE_PATH}
+    chgrp -R 0 "${INVENIO_INSTANCE_PATH}" && \
+    chmod -R g=u "${INVENIO_INSTANCE_PATH}"
```

Comment:
The `PATH` changes are affecting also the discoverability of the `cernopendata` executable.

I see that it is now installed in:

```
/opt/rh/rh-python36/root/usr/bin/cernopendata
```

Let's add `/opt/rh/rh-python36/root/usr/bin/` path to default `PATH` so that the old commands like:

```
docker exec -i -t opendatacernch_web_1 cernopendata fixtures records -f /code/cernopendata/modules/fixtures/data/records/cms-derived-csv-Run2011A.json --mode insert
```

will keep working.

Reply:
This is because we are using the RH software collections and the python bin folder won't be exported until `scl enable` is run...

I am adding it to the path, but, could we also just run everything through bash `docker exec -i -t opendatacernch_web_1 bash -c cernopendata` and don't rely on the specific path?

Reply:
Adding `bash -c ...` also works, please just edit `DEVELOPING.rst` to amend the example commands...